README.md

```diff
@@ -4,22 +4,19 @@
 
 **🌎 A forever evolving, interplanetary microblogging platform. 🚀**
 
-**Misskey** is a distributed microblogging platform with advanced features such as Reactions and a highly customizable UI.
-
-[Learn more](https://misskey-hub.net/)
+[**Misskey**](https://misskey-hub.net/en) is a distributed microblogging platform
+with advanced features such as reactions and a highly customizable UI.
 
 ---
 
-[✨ Find an instance](https://misskey-hub.net/instances.html)
+[✨ Find an instance](https://join.misskey.page)
 •
-[📦 Create your own instance](https://misskey-hub.net/docs/install.html)
+[📦 Create your own instance](https://misskey-hub.net/en/docs/install.html)
```

Comment:
I'm not sure if these links should be changed. Misskey Hub should redirect you to the English version based on your user agent's reported language. (Currently you will be redirected to the English version if your user agent does not report Japanese language.)

Reply:
That does not work for me, I get the Japanese versions and understand no word :/

Since this is an English Readme, I would expect links to default to English, too. People that do not prefer English probably won't find their introduction here, anyways.

Reply:
Since join.misskey.page is not a website managed by Misskey Dev, I think Misskey Hub is a better choice here.

Reply:
So changing the 2nd link is fine, but the 1st one should still go to misskey hub (in english).